Formula/sile.rb

```diff
@@ -155,7 +157,7 @@ def install
 
           system "luarocks", "make",
                              "#{r.name}-#{r.version}-1.rockspec",
-                             "ZLIB_DIR=#{Formula["zlib"].opt_prefix}",
+                             "ZLIB_DIR=#{MacOS.sdk_path_if_needed}/usr",
```

Comment:
Can you split this in an on_macos / on_linux block and keep the formula path for linux?